node/src/sub_lib/blockchain_bridge.rs

```diff
@@ -22,8 +23,8 @@ pub struct BlockchainBridgeConfig {
 #[derive(Clone, PartialEq, Eq)]
 pub struct BlockchainBridgeSubs {
     pub bind: Recipient<BindMessage>,
-    pub report_accounts_payable: Recipient<ReportAccountsPayable>,
-    pub request_balances_to_pay_payables: Recipient<RequestBalancesToPayPayables>,
+    pub report_accounts_payable: Recipient<OutcomingPaymentsInstructions>,
```

Comment:
In English, there are the words "outgoing" and "incoming." Everybody knows what these mean. "Outcoming" is not so clear. You came up with "outbound," which is not bad either. ("IBCD" stands for "Inbound Client Data," so there's precedent.) But not "outcoming," please.